src/porepy/applications/verification_setups/mandel_biot.py

```diff
@@ -1566,4 +1529,38 @@ class MandelSetup(  # type: ignore[misc]
     MandelDataSaving,
     BiotPoromechanics,
 ):
-    """Mixer class for Mandel's consolidation problem."""
+    """Mixer class for Mandel's consolidation problem.
+
+    Admissible model parameters at are:
```

Comment:
Perheps "Model parameters of special relevance to this class"? Because: 1 technically, more or less anything's admissible and 2 I think that other parts of the model (non-mandel-specific) access parameters not listed below.